select_command/_index.md

```diff
@@ -0,0 +1,60 @@
+
+# <center><b>Mission 1: Save the Planet of Fun From Boredom!</b></center>
+
+## <b> Task 1: Take a peek at the information on the Planet of Fun - Learn SELECT </b>
+
+<b>What the command does:</b> The SELECT command allows you to choose what data to show from a database.

```

Comment:
Can you try seeing if surrounding with ** (markdown syntax) instead of <b> tags make the text bold?

Reply:
It works! Thanks Oliver!